### PR TITLE
Bump elasticsearch and postgres packages

### PR DESCRIPTION
**Pull request by jaym**
The all the things to localhost change requires updated config for
elasticsearch and postgres. This brings in those changes

### CODE REVIEW

File: .expeditor/create-manifest.rb [hunk 1]
```diff
@@ -300,7 +300,7 @@ def parse_products_meta_file(path)
   # while still allowing the clients of these databases to upgrade their client
   # libraries if any fixes are shipped there.
   "automate-postgresql"    => {"origin" => "chef", "name" => "automate-postgresql",    "version" => "9.6.11", "release" => "20200723182209"},
```
Comment (stevendanna) [inline .expeditor/create-manifest.rb:302]: Do we neeed to do postgresql as well?

**Review by stevendanna — APPROVED** [review]
Thanks!

File: .expeditor/create-manifest.rb (hunk 1 above)
Comment (jaym) [reply .expeditor/create-manifest.rb:302]: let me double check

Comment (jaym) [reply .expeditor/create-manifest.rb]: the package was modified, so i brought it in

**Review by stevendanna — APPROVED** [review]
Double Approved!

**Review by yzl — APPROVED** [review]

**Pull request merged**